demos/esc/README.md

```diff
@@ -0,0 +1,85 @@
+# ESC Library Demo - eurus-project
+
+This is the example application demonstrating usage of the 
+Eurus Flight Controller (EFC) ESC (Electronic Speed Controller)
+library within the Zephyr framework.
+
+## Features & Demo configuration
+
+The library supports multiple ESC protocols, including:
+- Classic PWM
+- Oneshot 125
+- Oneshot 42
+- Multishot
+
+Default ESC protocol in this sample is PWM, but it can be reconfigured 
+using menuconfig. To start menuconfig, the demo needs to be built with: 
```

Comment:
In markdown, there is no need to wrap lines around a certain character count, the viewer will do that. Generally it is best to start new lines for new sentences and double newlines for paragraphs.